Adapter.C

```diff
@@ -611,7 +611,7 @@ void preciceAdapter::Adapter::adjustSolverTimeStep()
     // Update the solver's timestep (but don't trigger the adjustDeltaT(),
     // which also triggers the functionObject's adjustTimeStep())
     // TODO: Keep this in mind if any relevant problem appears.
-    const_cast<Time&>(runTime_).setDeltaT(timestepSolver_, false);
+    const_cast<Time&>(runTime_).setDeltaTNoAdjust(timestepSolver_);
```

Comment:
TODO: Update for target branch

Reply:
Are these TODOs for me? I don't understand

Reply:
These (TODO) changes come from the OpenFOAM6 branch, which we don't want to bring into develop/master. If you rebase on develop, they will get fixed automatically. ;-) I just added these comments so that we don't forget it.